keyboards/chidori/rules.mk

```diff
@@ -1,9 +1,5 @@
 # MCU name
 MCU = atmega328p
-ARCH = AVR8
-F_USB = $(F_CPU)
```

Comment:
Where does this get defined for 328p if not by the keyboard? I don't see these in mcu_selection.mk.

Reply:
`ARCH`, `F_USB` and the `INTERRUPT_CONTROL_ENDPOINT` define are LUFA-specific (technically, so is `F_CPU` but it's been added into all the usbconfigs so it's still controlled from rules.mk).